src/NServiceBus.Core/Endpoint.cs

```diff
@@ -1,21 +1,45 @@
 namespace NServiceBus
 {
     using System.Threading.Tasks;
+    using ObjectBuilder;
 
     /// <summary>
     /// Provides factory methods for creating and starting endpoint instances.
     /// </summary>
     public static class Endpoint
     {
+        /// <summary>
+        /// Prepares an endpoint to run with and externally managed container.
+        /// </summary>
+        /// <param name="configuration">The endpoint configuration.</param>
+        /// <param name="configureComponents">The registration api adapter for the external container.</param>
+        /// <returns>The prepared endpoint.</returns>
```

Comment:
The xml comment isn't reflecting the Prepare -> Configure name change.

Reply:
Fixed